src/NHSD.BuyingCatalogue.Ordering.Api/Models/UpdateOrderItemModel.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.Models
+{
+    public class UpdateOrderItemModel
+    {
+        public string DeliverDate { get; set; }
```

Comment:
DeliveryDate not DeliverDate

Reply:
This hasn't been changed, from what I can tell

Reply:
I thought i got this let me check